docs/content/latest/integrations/change-data-capture/cdc-generic.md

```diff
@@ -1,61 +1,60 @@
 ---
-title: Explore change data capture (CDC) on macOS
-headerTitle: Change data capture (CDC)
-linkTitle: Change data capture (CDC)
-description: Use a local YugabyteDB cluster (on macOS) to stream data changes to stdout using the CDC API.
+title: Change Data Capture
+headerTitle: Change Data Capture
+linkTitle: Change Data Capture
+description: Use a local YugabyteDB cluster to stream data changes to stdout using the Change Data Capture API.
 beta: /latest/faq/general/#what-is-the-definition-of-the-beta-feature-tag
 aliases:
   - /latest/deploy/cdc/cdc-to-stdout/
   - /latest/explore/change-data-capture/
   - /latest/explore/change-data-capture-macos/
+  - /latest/explore/change-data-capture-linux/
 menu:
   latest:
-    identifier: change-data-capture-1-macos
-    parent: explore
+    identifier: change-data-capture-1-cdc-generic
+    parent: integrations
     weight: 610
 isTocNested: true
 showAsideToc: true
 ---
 
 <ul class="nav nav-tabs-alt nav-tabs-yb">
-
   <li >
-    <a href="/latest/explore/change-data-capture/macos" class="nav-link active">
-      <i class="fab fa-apple" aria-hidden="true"></i>
-      macOS
+    <a href="{{< relref "./cdc-generic.md" >}}" class="nav-link active">
+      Generic
     </a>
   </li>
-
   <li >
-    <a href="/latest/explore/change-data-capture/linux" class="nav-link">
-      <i class="fab fa-linux" aria-hidden="true"></i>
-      Linux
+    <a href="{{< relref "./cdc-kafka.md" >}}" class="nav-link">
+      Kafka
     </a>
   </li>
 
 </ul>
 
-[Change data capture (CDC)](../../../architecture/cdc-architecture) can be used to asynchronously stream data changes from a YugabyteDB cluster to external systems like message queues and OLAP warehouses. The data changes in YugabyteDB are detected, captured, and then output to the specified target.  In the steps below, you will use a local YugabyteDB cluster to stream data changes to `stdout` using the CDC API.
+[Change data capture (CDC)](../../../architecture/cdc-architecture) can be used to asynchronously stream data changes from a YugabyteDB cluster to external systems such as message queues and OLAP warehouses. The data changes in YugabyteDB are detected, captured, and then directed to a specified target.
 
```

Comment:
link is now actually architecture/docdb-replication/change-data-capture/ kept alive by an alias I guess

Reply:
changed